src/Features/Core/Portable/LanguageServices/SymbolDisplayService/AbstractSymbolDisplayService.AbstractSymbolDescriptionBuilder.cs

```diff
@@ -290,7 +290,15 @@ private async Task AddDescriptionPartAsync(ISymbol symbol)
                 }
                 else if (symbol is INamedTypeSymbol namedType)
                 {
-                    await AddDescriptionForNamedTypeAsync(namedType).ConfigureAwait(false);
+                    if (namedType.IsTupleType)
+                    {
+                        AddToGroup(SymbolDescriptionGroups.MainDescription,
+                            ToDisplayParts(symbol, s_descriptionStyle));
+                    }
+                    else
+                    {
+                        await AddDescriptionForNamedTypeAsync(namedType).ConfigureAwait(false);
+                    }
```

Comment:
why as this change needed?

Reply:
I hope I'm remembering correctly. If you look inside `AddDescriptionForNamedTypeAsync`, it gets the original definition (`symbol.OriginalDefinition`), prints that, then prints type arguments.
For tuples, we want to print the tuple syntax instead.

Previously, tuple symbols were original definitions, but that is no longer always the case.

---
In reply to: [355809677](https://github.com/dotnet/roslyn/pull/39370#discussion_r355809677) [](ancestors = 355809677)

Reply:
I hope I'm remembering correctly. If you look inside `AddDescriptionForNamedTypeAsync`, it gets the original definition (`symbol.OriginalDefinition`), prints that, then prints type arguments.
For tuples, we want to print the tuple syntax instead.

Previously, tuple symbols were original definitions, but that is no longer always the case.